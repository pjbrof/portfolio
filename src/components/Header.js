import React from 'react';

export default class Header extends React.Component {

  addTwoNumbers(a, b) {
    return a + b;
  }

  render() {
    return (
      <div className="header">
        <h1>Header</h1>
      </div>
    );
  }
}
