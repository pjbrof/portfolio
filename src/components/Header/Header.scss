.header {
  background-color: #f2f2f2;
  width: 100%;
}

.header h1 {
  margin: 0;
  padding: 20px;
}
