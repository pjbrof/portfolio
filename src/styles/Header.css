.header {
  background-color: #EFEFEF;
  height: 50px;
	width: 100%;
}
