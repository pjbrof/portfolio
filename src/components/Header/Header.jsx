import React from 'react';
import { connect } from 'react-redux';

import { fetchInfo } from '../../actions/dataActions';

import './Header.scss';

// eslint-disable-next-line react/prefer-stateless-function
class Header extends React.Component {
  render() {
    return (
      <div className="header">
        <h1>Header</h1>
      </div>
    );
  }
}

const mapStateToProps = store => store.data;

const mapActionsToProps = {
  fetchInfo
};

export default connect(
  mapStateToProps,
  mapActionsToProps
)(Header);
