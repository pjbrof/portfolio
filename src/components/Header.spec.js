import React from 'react';
import renderer from 'react-test-renderer';
import Header from './Header';

describe('<Header />', () => {
    it('Renders the header component', () => {
        const tree = renderer
        .create(<Header />)
        .toJSON();
        expect(tree).toMatchSnapshot();
    });

    test('is this a suite?', () => {
        const header = new Header();
        const result = header.addTwoNumbers(1,2);
        expect(result).toEqual(3);
    });
});
