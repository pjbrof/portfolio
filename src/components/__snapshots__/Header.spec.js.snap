// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> Renders the header component 1`] = `
<div
  className="header"
>
  <h1>
    Header
  </h1>
</div>
`;
